packages/pangea-sdk/tests/integration/test_authn.py

```diff
@@ -255,7 +265,7 @@ def test_authn_c2_login_n_client_session_invalidate(self):
             token = response_login.result.active_token.token
 
             # list sessions
-            response = self.authn.client.session.list(token=token)
+            response = self.authn.client.session.list(token=token, filter={})
```

Comment:
Are these empty filter dicts necessary now?

Reply:
It's optional, it was just for testing that also accept a dict, even if it's empty